tofhir-server/src/main/scala/io/tofhir/server/endpoint/ConceptMapEndpoint.scala

```diff
@@ -68,6 +77,66 @@ class ConceptMapEndpoint(conceptMapRepository: IConceptMapRepository) extends La
     }
   }
 
+  /**
+   * Route to update concept map csv header
+   * @param terminologyId terminology id
+   * @param conceptMapId concept map id
+   * @return
+   */
+  private def updateConceptMapHeader(terminologyId: String, conceptMapId: String): Route = {
```

Comment:
Can you please follow the same convention while naming the functions inside an Endpoint class?